src/Commands/GenerateModule.php

```diff
@@ -0,0 +1,321 @@
+<?php
+
+namespace FOP\Console\Commands;
+
+use FOP\Console\Command;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+/**
+ * This command is a working exemple.
+ */
+final class GenerateModule extends Command
+{
+    protected static $tabHook=array();
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:console:generate-module')
```

Comment:
The command should be fop:module:generate

Reply:
ok